model-optimizer/extensions/middle/UnsqueezeTileReshapeBlockToInterpolate.py

```diff
@@ -76,79 +89,104 @@ def pattern(self):
             ]
         )
 
-    def replace_pattern(self, graph: Graph, match: dict):
+    @staticmethod
+    def is_applicable(match: dict) -> bool:
         unsqueeze_node = match['unsqueeze']
```

Comment:
Please, add comments to the code (basically copy-paste parts of the code from the PR description)

Reply:
Done.